genieutils.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b4b3abb..760a912 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -12,11 +12,11 @@ project(genieutils)
+ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake/)
+ #set(CMAKE_CXX_FLAGS "-std=gnu++0x")
+ 
+-#set(Boost_USE_STATIC_LIBS        ON)
++set(Boost_USE_STATIC_LIBS        ON)
```

Comment:
Is that a tested change? I remember we had problems because of this @HSZemi when statically compiling, no?

Reply:
Also why we need to statically link now as a standard?

Reply:
This patch is already in the `patchGenieutils.sh` file found in master. I simply executed the bash script and generated this patch file with `git diff` so the end result should be the same

Reply:
True, it's less a review comment then a question to Hszemi, if it's still said that the project should be statically linked in general and not just for special purposes.

Reply:
I need a statically linked binary that I can copy \~wherever\~ in order to automatically update the random costs mods daily.
I currently see no benefit in _additionally_ maintaining a dynamically linked version :shrug: